sdk/core/core-xml/package.json

```
@@ -70,7 +70,7 @@
   "prettier": "@azure/eslint-plugin-azure-sdk/prettier.json",
   "dependencies": {
     "tslib": "^2.2.0",
-    "fast-xml-parser": "^4.0.8"
+    "fast-xml-parser": "^4.2.2"
```

Don't we need 4.2.4 for the fix?

```suggestion
    "fast-xml-parser": "^4.2.4"
```

---

Not sure what happened there, updated version